examples/guestbook-go/README.md

```diff
@@ -88,97 +93,106 @@ Once that's up you can list the pods in the cluster, to verify that the master a
-redis-slave-3psic    10.244.0.4                                            kubernetes-minion-mluf/104.197.10.10     name=redis,role=slave    Running   3 minutes    
-                                  redis-slave    gurpartap/redis                                                                             Running   3 minutes    
-redis-slave-qtigf    10.244.2.4                                            kubernetes-minion-rcgd/130.211.122.180   name=redis,role=slave    Running   3 minutes    
-                                  redis-slave    gurpartap/redis                                                                             Running   3 minutes   
+$ kubectl get pods
+NAME                                           READY     REASON    RESTARTS   AGE
+guestbook-3crgn                                1/1       Running   0          2m
+guestbook-gv7i6                                1/1       Running   0          2m
+guestbook-x405a                                1/1       Running   0          2m
+redis-master-xx4uv                             1/1       Running   0          23m
+redis-slave-b6wj4                              1/1       Running   0          6m
+redis-slave-iai40                              1/1       Running   0          6m
+... 
 ```
 
 You will see a single redis master pod, two redis slaves, and three guestbook pods.
 
 ### Step Six: Create the guestbook service.
 
-Just like the others, you want a service to group your guestbook pods.  The service specification for the guestbook is in `examples/guestbook-go/guestbook-service.json`.  There's a twist this time - because we want it to be externally visible, we set `"type": "LoadBalancer"` for the service.
+Just like the others, you want a service to group your guestbook pods.  The service specification for the guestbook is in [examples/guestbook-go/guestbook-service.json](guestbook-service.json).  There's a twist this time - because we want it to be externally visible, we set `"type": "LoadBalancer"` for the service.
 
 ```shell
 $ kubectl create -f examples/guestbook-go/guestbook-service.json
 
-$ kubectl get services
-NAME           LABELS                   SELECTOR                 IP(S)          PORT(S)
-guestbook      name=guestbook           name=guestbook           10.0.114.109   3000/TCP
-redis-master   name=redis,role=master   name=redis,role=master   10.0.11.173    6379/TCP
-redis-slave    name=redis,role=slave    name=redis,role=slave    10.0.234.24    6379/TCP
-```
+      An external load-balanced service was created.  On many platforms (e.g. Google Compute Engine),
+      you will also need to explicitly open a Firewall rule for the service port(s) (tcp:3000) to serve traffic.
 
-To play with the service itself, find the external IP of the load balancer:
+      See https://github.com/GoogleCloudPlatform/kubernetes/tree/master/docs/services-firewall.md for more details.
```

Comment:
Should use relative linking here -> [Services and Firewalls](../../docs/services-firewall.md)